src/components/AddMovie/AddMovie.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import Dialog from '../Dialog/Dialog';
+import MovieForm from '../MovieForm/MovieForm';
+import { AddMovieProps } from '../../models';
+
+const AddMovie = ({ onSubmit }: AddMovieProps) => {
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+
+  const handleClose = () => {
+    setIsDialogOpen(false);
+  };
+
+  return (
+    <>
+      <button onClick={() => setIsDialogOpen(true)}>Add Movie</button>
+      {isDialogOpen && (
+        <Dialog title={'Add Movie'} onClose={handleClose}>
```

Comment:
Please stick to one notation, it should be `att={'string'}` everywhere else or `title="Add Movie"` here.